### ISSUE TITLE
Paste event/line break issue. Cant use delete button after paste.

### ISSUE THREAD

**Issue by Engoldo** [JiHong88/SunEditor#767]
Hi, could someone help me to understand paste event issue.
By default we have "/n" symbol in textarea.
![image](https://user-images.githubusercontent.com/25320389/119940191-8064f900-bf97-11eb-89fb-d1b6eee982d8.png)
When we just inputing text "/n" will be removed.
![image](https://user-images.githubusercontent.com/25320389/119940446-db96eb80-bf97-11eb-811c-6795183faf56.png)
But if we pasting text inside text area "/n" will be there too. And if "/n" symbol there you cant use ctrl+a and delete button to delete text from text area. 
![image](https://user-images.githubusercontent.com/25320389/119940690-3b8d9200-bf98-11eb-85d4-e066fec4f83b.png)

Maybe I missed kind of an option for config?

**Comment by JiHong88** [comment]
> But if we pasting text inside text area "/n" will be there too. And if "/n" symbol there you cant use ctrl+a and delete button to delete text from text area.

`\n` is the `<br>` tag.
To position the cursor, this tag is always maintained when there is no text.
If you mean the `<br>` tag won't be deleted, this is the intended behavior.

**Comment by Engoldo** [comment]
Yeah, I tried to remove <br> tag. But in that case delete button doesn't work too. I tried to paste some text, press ctrl+a and trying to delete text and I cant.

**Comment by JiHong88** [comment]
Doesn't the text "Hello" get deleted?
This issue is not reproduced.
Please tell me the test environment and the exact test method.

**Comment by Engoldo** [comment]
Config:
![image](https://user-images.githubusercontent.com/25320389/120189486-1743e680-c220-11eb-97fa-60358d160248.png)
![image](https://user-images.githubusercontent.com/25320389/120189527-232fa880-c220-11eb-9297-dc9e93a48e99.png)
Template:
![image](https://user-images.githubusercontent.com/25320389/120189607-3cd0f000-c220-11eb-9eb5-bd2ce34633a6.png)
I found that issue manually. Also I reproduced it in examples - http://suneditor.com/sample/html/examples.html#image
I removed the placeholder text, copied the text from the address bar, put it in textarea, pressed ctrl+a and tried to remove it with delete button. And it doesn't remove.
